operator/Makefile

```diff
@@ -3,26 +3,26 @@
 
 include ../Makefile.defs
 
-# Build all ipam managers in development environments
-ifeq ($(PKG_BUILD),)
-	EXTRA_GO_BUILD_FLAGS += -tags ipam_provider_aws,ipam_provider_azure,ipam_provider_operator
-endif
+TARGETS := cilium-operator cilium-operator-generic cilium-operator-aws cilium-operator-azure
```

Comment:
All targets will be built by default, hope that satisfies what `ifeq ($(PKG_BUILD),)`  did previously.